python/lsst/ts/salobj/topics/mock_write_topic.py

```diff
@@ -0,0 +1,91 @@
+from __future__ import annotations
+
+# This file is part of ts_salobj.
+#
+# Developed for the Rubin Observatory Telescope and Site System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+__all__ = ["MockWriteTopic"]
+
+import typing
+
+from .. import type_hints
+from .write_topic import MAX_SEQ_NUM, WriteTopic
+
+if typing.TYPE_CHECKING:
+    from ..sal_info import SalInfo
+
+
+class MockWriteTopic(WriteTopic):
+    r"""A version of WriteTopic that records every message written
```

Comment:
Maybe I miss something but why is there an `r` in front of the `"""`?

Reply:
I'll remove it. When a string has back-slashes in it, it avoids complaints about unrecognized escape sequences (without the need to double each backslash).